### PR TITLE
Civ 5037

### PR DESCRIPTION
**Pull request by kamilewandowski**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-5037

### Change description ###
adding welsh translation for send-your-response-by-email

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -1515,7 +1515,44 @@
       "EXAMPLE": "Er enghraifft cyfraniadau Yswiriant Gwladol, Treth Incwm neu TAW."
     },
     "SEND_YOUR_RESPONSE_BY_EMAIL": {
-      "PAGE_TITLE": "Send your response by email"
+      "PAGE_TITLE": "Send your response by email",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:1518]: alphabetical order

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -1515,7 +1515,44 @@
...
+      "HEADER": "Gwrthod yr hawliad cyfan a gwneud gwrth-hawliad",
+      "COUNTER_CLAIM_CHOSEN": "Rydych wedi dewis gwneud gwrth-hawliad. Mae hyn yn golygu ni all eich amddiffyniad barhau ar-lein. Dilynwch y camau hyn i lwytho, llenwi a dychwelyd ffurflen N9B. Rhaid i chi dalu ffi'r llys. Peidiwch â chreu hawliad newydd.",
+      "WE_MUST_RECEIVE_COMPLETED_FORM": "Rhaid i ni gael eich ffurflen wedi'i llenwi cyn 4pm ar 20 Awst 2023. Bydd yr hawliad yn eich erbyn yn mynd yn ei flaen os na fyddwn yn cael eich ffurflen.",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:1521]: `4pm ar 20 Awst 2023` this is dynamic date, see how it is done in en.json `{{ deadline }}`

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -1515,7 +1515,44 @@
...
+      },
+      "STEP_2": "Cam 2: E-bostio neu bostio'r ffurflen N9B",
+      "EMAIL_COMPLETED": "E-bostio'r ffurflen N9B wedi'i llenwi i: contactocmc@justice.gov.uk neu ymholiadaucymraeg@justice.gov.uk",
+      "SEND_BY_POST": "Neu, gallwch anfon y ffurflen drwy'r post i:",
+      "STEP_3": "Cam 3: Talu'r ffi llys",
+      "PAY_COURT_FEE": "Bydd angen i chi dalu ffi'r llys i wneud gwrth-hawliad. Bydd y llys yn cysylltu â chi i gymryd y taliad.",
+      "DO_NOT_CREATE_NEW_CLAIM": "Peidiwch â chreu hawliad newydd os ydych eisiau gwneud gwrth-hawliad. Defnyddiwch ffurflen N9B.",
+      "HELP_AND_SUPPORT": "Cymorth a Chefnogaeth",
+
+      "PDF_ADMISSION": "Download admission form (PDF, 102 KB)",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:1539]: If we don't have these translations ticket should still be blocked we don't want to address tech debt ticket with another tech debt

File: src/main/modules/i18n/locales/cy.json [hunk 4]
```diff
@@ -1692,7 +1729,12 @@
       "WHAT_LANGUAGE": "What language needs to be interpreted?"
     },
     "UNDER_18": {
-      "PAGE_TITLE": "Under 18"
+      "PAGE_TITLE": "Under 18",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:1732]: aren't these part of the different tech debt? CIV-5036

**Review by commitBlob — CHANGES_REQUESTED** [review]

**Comment by commitBlob** [comment]
removed deployment labels until ticket is ready for QA

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json]: fixed

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json]: updated that

File: src/main/modules/i18n/locales/cy.json (hunk 4 above)
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json:1731]: yes, it is civ-5036 I just created this branch from the previous

File: src/main/modules/i18n/locales/cy.json (hunk 3 above)
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json]: translations in place

File: src/main/modules/i18n/locales/cy.json [hunk 5]
```diff
@@ -1,4 +1,13 @@
 {
+  "Claim number": "Mialc rebmun",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:2]: These are not translations it is just text written in reverse.
You should remove all translations until common object (line 11)

**Review by commitBlob — COMMENTED** [review]
When you are working on the ticket please branch from master.
We want to have one branch and one PR per ticket

File: src/main/modules/i18n/locales/cy.json (hunk 5 above)
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json]: fixed in next commits

File: src/main/modules/i18n/locales/cy.json [hunk 6]
```diff
@@ -1484,7 +1493,43 @@
       "EXAMPLE": "Er enghraifft cyfraniadau Yswiriant Gwladol, Treth Incwm neu TAW."
     },
     "SEND_YOUR_RESPONSE_BY_EMAIL": {
-      "PAGE_TITLE": "Send your response by email"
+      "COUNTER_CLAIM_CHOSEN": "Rydych wedi dewis gwneud gwrth-hawliad. Mae hyn yn golygu ni all eich amddiffyniad barhau ar-lein. Dilynwch y camau hyn i lwytho, llenwi a dychwelyd ffurflen N9B. Rhaid i chi dalu ffi'r llys. Peidiwch â chreu hawliad newydd.",
```
[the remark this answers is not in the document]
Comment (kamilewandowski) [reply src/main/modules/i18n/locales/cy.json]: fixed

**Review by commitBlob — APPROVED** [review]

**Pull request merged**